optuna/integration/allennlp/_pruner.py

```diff
@@ -53,9 +52,9 @@ def wrapper(subclass: Any, *args: Any, **kwargs: Any) -> Any:
 
 
 def _create_pruner(
-    pruner_class: Optional[str],
```

Comment:
Maked `pruner_*` non-optional because `_create_pruner` is called inside `AllenNLPPruningCallback` and it should raises an exception when `pruner_*` is not available.

https://github.com/optuna/optuna/blob/eb185c25232849e30f3e4ff218fd28ebb27c2f88/optuna/integration/allennlp/_pruner.py#L175-L179

https://github.com/optuna/optuna/blob/eb185c25232849e30f3e4ff218fd28ebb27c2f88/optuna/integration/allennlp/_variables.py#L63-L73